velox/exec/tests/utils/ArbitratorTestUtil.h

```diff
@@ -86,6 +86,27 @@ struct TestAllocation {
   }
 };
 
+/// Scoped abort percentage utility that allows user to set the behavior of
+/// triggered abort.
+/// 'abortPct' indicates the chance of triggering abort. 100% means abort will
+/// always be triggered.
+/// 'maxInjections' indicates the max number of actual triggering, e.g. when
+/// 'abortPct' is 20 and 'maxInjections' is 10, continuous calls to
+/// testingTriggerAbort() will keep rolling the dice that has a chance of 20%
+/// triggering until 10 triggers have been invoked.
+class TestScopedAbortInjection {
+ public:
+  explicit TestScopedAbortInjection(
+      int32_t abortPct,
+      int32_t maxInjections = std::numeric_limits<int32_t>::max());
+
+  ~TestScopedAbortInjection();
+};
+
+/// Test utility that returns true if triggered abortion is evaluated to happen,
+/// false otherwise.
+bool testingTriggerAbort();
```

Comment:
```
s/testingTriggerAbort/testingMaybeTriggerAbort/
```
We can just trigger query abort inside this function: testingMaybeTriggerAbort(task)

Reply:
Got it, thanks.